keyboards/mkh_studio/bully/bully.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
+	K0A, K0B, K0C, K0D, K0E, K0F, K0G, K0H, K0I, K0J, K0K, K0L, K3L, \
+    K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I, K1J, K1K, K1L,      \
+    K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, K2I, K2J, K2K, K2L,      \
+    K3A, K3B, K3C, K3D,      K3F, K3G,      K3I, K3J, K3K            \
+) { \
+	{ K0A, K0B, K0C, K0D, K0E, K0F, K0G, K0H, K0I, K0J, K0K, K0L }, \
+	{ K1A, K1B, K1C, K1D, K1E, K1F, K1G, K1H, K1I, K1J, K1K, K1L }, \
+	{ K2A, K2B, K2C, K2D, K2E, K2F, K2G, K2H, K2I, K2J, K2K, K2L }, \
+	{ K3A, K3B, K3C, K3D, XXX, K3F, K3G, XXX, K3I, K3J, K3K, K3L } \
+}
```

Comment:
This file and `bully.c` should be removed, assuming the `matrix` data in info.json file is correct.

Reply:
Validated, done